.github/workflows/check.yml

```diff
@@ -18,32 +18,49 @@ jobs:
         shell: bash
     strategy:
       matrix:
+        installer: [pip]
         system:
           - ubuntu-latest
           - macos-latest
           - windows-latest
+          - macos-13
         version:
           - '1.0.1'
           - '2.18.1'
+          - '3.61.0'
+        include:
+        - system: ubuntu-22.04-arm
+          version: '3.61.0'
+          installer: uv
+        - system: ubuntu-22.04-arm
+          version: '3.61.0'
+          installer: pip
+        - system: windows-latest
+          version: '3.61.0'
+          installer: uv
     runs-on: ${{ matrix.system }}
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v3
-        if: matrix.system == 'windows'
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        if: ${{ matrix.system == 'windows' || contains(matrix.system, 'arm') }}
         with:
           python-version: '3.x'
+      - uses: astral-sh/setup-uv@v6
+        if: matrix.installer == 'uv'
       - run: npm ci
       - run: npm run lint
       - run: npm run test
       - name: local action with ${{ matrix.version }}
         uses: ./
         with:
           version: ${{ matrix.version }}
+      - run: dvc version || true
```

Comment:
I added this to assist with future debugging, but some packages are outdated and incompatible with the latest Python release, causing failures. I had to suppress it if it failed.